CloacaNative/IO/DataTypes/PyTextIOBase.cs

```diff
@@ -21,6 +21,12 @@ protected PyTextIOBaseClass(CodeObject __init__)
         private static PyTextIOBaseClass __instance;
         public static PyTextIOBaseClass Instance => __instance ?? (__instance = new PyTextIOBaseClass(null));
 
+        [ClassMember]
+        public static void test()
```

Comment:
I think a class member will need a PyObject self argument in order to work--even if you don't intend to use it. Consider it like you are writing it in Python:

def test(self):
  ...

Reply:
Ok, i was unsure about this since the Python specs seem to suggest that you can't instantiate these base classes - i had that in initially and removed it :{ i'll add them back